nuxeo-features/rest-api/nuxeo-rest-api-test/src/test/java/org/nuxeo/ecm/restapi/test/EmptyDocumentTest.java

```diff
@@ -165,4 +168,22 @@ public void testOverrideEmptyDocumentListenerValues() throws IOException {
             assertEquals("dummy subject", subjects.get(0).textValue());
         }
     }
+
+    @Test
+    @ConsoleLogLevelThreshold("ERROR")
+    public void testEmptyDocumentCreatioAndCallGetDocumentParentOperation() throws IOException {
+        //create the empty doc with parent
```

Comment:
Not sure if this comment is very useful here, in facr the line after just creates the parent folder, not the empty doc itself.